articles/other/other-ref-pricing-tool-bugs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Known bugs with the UKCloud Pricing Estimator Tool | UKCloud Ltd
+description: Known bugs with the UKCloud Pricing Estimator Tool
+services: other
+author: Dan Baker
+
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Known bugs with the UKCloud Pricing Estimator Tool
+toc_fullpath: Reference/other-ref-pricing-tool-bugs.md
+toc_mdlink: other-ref-pricing-tool-bugs.md
+---
+
+# Known bugs in the Pricing Estimator Tool
+
+This is an accompanying article to the [Pricing Estimator Tool guide](other-how-use-pricing-tool.md). This article intends to be a constantly updated list of issues and bugs users experience will the tool. As we update the tool, this article will be updated accordingly.
+
+## Current version
+
+The current version of the tool:
+
+- **Windows version**: 10.2.8
+
+- **Mac version**: 10.2.8
+
+## Known issues
+
+### Windows
+
+1. Within the UKCloud for VMware section, a user can add additional storage GiB without selecting the Storage type – with no error message.
+
+This storage is added to the estimate BUT the calculation does not work. It displays the additional storage as being free, which is incorrect.
+
+Workaround: if the user selects the storage type first and not enter any GiB then an error message appears which is correct.
```

Comment:
Workaround: When including additional storage in your estimate, be sure to select the storage type first. If you select a storage type but do not enter a storage amount, an error message is displayed.